modules/Microsoft.Storage/storageAccounts/.test/minPrem/deploy.test.bicep

```diff
@@ -38,7 +38,6 @@ module testDeployment '../../deploy.bicep' = {
   params: {
     enableDefaultTelemetry: enableDefaultTelemetry
     name: '<<namePrefix>>${serviceShort}001'
-    allowBlobPublicAccess: false
-    storageAccountAccessTier: 'Premium'
+    storageAccountSku: 'Premium_LRS'
```

Comment:
In a concurrent PR I actually removed this entire test. Do you see any value in it?

Reply:
Not really. Would however recommend keeping the parameter for access tier as guidance since we fell into the trap of it for our test file

Reply:
Agreed

Reply:
I've removed the test file and updated doco.